diff -urN a/host/host_linux_ppc64le.go b/host/host_linux_ppc64le.go
--- a/host/host_linux_ppc64le.go	1970-01-01 00:00:00.000000000 +0000
+++ b/host/host_linux_ppc64le.go	2025-10-01 00:00:00.000000000 +0000
@@ -0,0 +1,36 @@
+//go:build linux && ppc64le
+// +build linux,ppc64le
+
+package host
+
+const sizeOfUtmp = 384
+
+type utmp struct {
+	Type      int16
+	_         [2]byte
+	Pid       int32
+	Line      [32]byte
+	Id        [4]byte
+	User      [32]byte
+	Host      [256]byte
+	Exit      exit_status
+	Session   int32
+	Tv        timeval
+	Addr_v6   [4]int32
+	_         [20]byte
+}
+
+type exit_status struct {
+	Termination int16
+	Exit        int16
+}
+
+type timeval struct {
+	TvSec  int64
+	TvUsec int64
+}
+
+type timespec struct {
+	Sec  int64
+	Nsec int64
+}
